vnext/Microsoft.ReactNative.IntegrationTests/ReactNotificationServiceTests.cpp

```diff
@@ -247,6 +293,36 @@ TEST_CLASS (ReactNotificationServiceTests) {
     subscription = nullptr;
     TestCheck(!s.IsSubscribed());
   }
+
+  TEST_METHOD(NotificationBetweenAppAndModule) {
+    TestEventService::Initialize();
+
+    auto reactNativeHost =
+        TestReactNativeHostHolder(L"ReactNotificationServiceTests", [](ReactNativeHost const &host) noexcept {
```

Comment:
I realize these are integration tests, should we have some unittests for this as well, especially to test things like:
* subscribe, unsubscribe, subscribe
* nested notification services etc
* subscribing twice on the same node
* subscribing twice on sibblings and then unsubscribe.
* Error case for removing subscription twice?

Reply:
Good point! Some of these tests are already there: see other tests in the same test file. I want to address the current issue as soon as possible, and I will address the additional test scenarios in another PR. I have created the issue #6881 for these items to be on my radar.

Reply:
👍

---
In reply to: [557835279](https://github.com/microsoft/react-native-windows/pull/6877#discussion_r557835279) [](ancestors = 557835279)